internal/armadactl/queue.go

```diff
@@ -68,6 +81,44 @@ func (a *App) GetQueue(name string) error {
 	return nil
 }
 
+func (a *App) getAllQueuesAsAPIQueue(queueNames []string, labels []string, inverse bool) ([]*api.Queue, error) {
+	queueContainsAllLabelsAndInProvidedNames := func(q *api.Queue) bool {
```

Comment:
Do we want/need to support a user giving both queue names and queue labels.  The logic here would be simpler if we restricted them to either queue names or queue labels.  I'd argue unless we have a good use case in mind we should stick to the simpler implementation and add something more complex if we find we need it.